examples/third_party/mesa/BUILD.libdrm.bazel

```diff
@@ -0,0 +1,26 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "meson")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["**"]),
+)
+
+meson(
+    name = "libdrm",
+    lib_source = ":all_srcs",
+    out_lib_dir = "lib/x86_64-linux-gnu",
```

Comment:
Again this should probably be behind a select on the platform and arch but this is good enough for now.

Reply:
I'll leave this as is